spatial-index-h3/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>CARTO for deck.gl - H3 Spatial Index 12M rows</title>
+  </head>
+  <body>
+    <div id="map"></div>
+    <canvas id="deck-canvas"></canvas>
+    <div id="story-card">
+      <p class="overline">✨👀 You're viewing</p>
+      <h2>CARTO for deck.gl - H3 Spatial Index 12M rows</h2>
+      <p>
+        More info about
+        <a
+          href="https://docs.carto.com/carto-user-manual/workflows/components/spatial-indexes"
+          rel="noopener noreferrer"
+          target="_blank"
+          >H3 Spatial Index</a
```

Comment:
Link to https://carto.com/blog/carto-spatial-features-urbanity-climatology-elevation